### PR TITLE
Feature: update criteria for user verification

### PR DESCRIPTION
**Pull request by ShellyXueHan**

### CODE REVIEW

File: src/libs/sso-utils.js [hunk 1]
```diff
@@ -37,6 +37,26 @@ export const checkCredentialValid = credentials => {
 export const checkUserProfile = userInfo =>
   !(!userInfo.email || !userInfo.firstName || !userInfo.lastName);
 
+export const checkArray = array => !Array.isArray(array) || !array.length;
+
+export const checkRocketChatSchema = async userInfo => {
```
Comment (patricksimonian) [inline src/libs/sso-utils.js]: best to have a js doc on this function,

Comment (patricksimonian) [inline src/libs/sso-utils.js]: does this func need to be async? doesn't seem to have anything async about it

File: src/libs/sso-utils.js [hunk 2]
```diff
@@ -196,31 +216,34 @@ export const getUserInfoById = async (credentials, id) => {
...
-    }
-    // TODO: add check on authorization by token
+    // if user has complete profile and matches requirement, return the current sso group status:
+    return {
+      ...accountStatus,
+      ...{
+        isPending: ssoGroupNames.includes('pending'),
+        isAuthorized: ssoGroupNames.includes('registered'),
+      },
+    };
```
Comment (patricksimonian) [inline src/libs/sso-utils.js:220]: looks good but the above is equivalent to 
```js
return {
      ...accountStatus,
       isPending: ssoGroupNames.includes('pending'),
       isAuthorized: ssoGroupNames.includes('registered'),
    };

```

as an fyi

**Review by patricksimonian — APPROVED** [review]

File: src/libs/sso-utils.js (hunk 1 above)
Comment (ShellyXueHan) [reply src/libs/sso-utils.js]: yes I forgot to await for `isUserInOrgs`

File: src/libs/sso-utils.js (hunk 2 above)
Comment (ShellyXueHan) [reply src/libs/sso-utils.js:220]: haha yea, updated!

**Pull request merged**